### PR TITLE
[RFR]Added ouiaid attributes for ui components

### PR DESCRIPTION
**Pull request by mayurilahane**

### CODE REVIEW

File: src/routes/InventoryDetail.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ const Inventory = ({ entity, currentApp, clearNotifications, loadEntity }) => {
             }}
         >
             <PageHeader className={classnames('pf-m-light ins-inventory-detail', additionalClasses)} >
-                <Breadcrumb>
+                <Breadcrumb ouiaId="inventory">
```
Comment (mayurilahane) [inline src/routes/InventoryDetail.js:58]: Not sure about this!
Whether to put here or in breadcrumb_items or its not required

Comment (karelhala) [reply src/routes/InventoryDetail.js:58]: I would change it from `inventory` to `systems-list` as that's what the previous page shows.

File: src/components/DeleteModal.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ const DeleteModal = ({ handleModalToggle, isModalOpen, currentSytem, onConfirm }
         </Split>
         <Level hasGutter>
             <LevelItem>
-                <Button variant="danger" onClick={onConfirm}>
+                <Button variant="danger" onClick={onConfirm} ouiaId="Remove">
```
Comment (karelhala) [inline src/components/DeleteModal.js:53]: I'd change the ouiaId to be more specific `Remove` to `confirm-delete` as that's what this button does.

**Comment by karelhala** [comment]
@mayurilahane  thanks for opening this PR! If you need any help, feel free to ping me and I'll be happy to help you. Do you have other places where you want to add ouia attrs?

**Comment by mayurilahane** [comment]
I have almost searched in all files but I didn't get any other components to change!

Thanks for the help :)

**Review by karelhala — APPROVED** [review]

**Pull request merged**